File: src/main/swift/Sources/swift/fr/jhelp/kotlinLightSamples/WhileSample.swift

public func testWhile()
{
    var count = 1

    while count < 1000
    {
        print("While count=>\(count)")
        count *= 2
    }
}

public func testDoWhile()
{
    var count = 1

    repeat
    {
        print("Do while count=>\(count)")
        count *= 2
    }
    while count < 1000
}

File: src/main/kotlin/fr/jhelp/kotlinLightSamples/WhileSample.kt
package fr.jhelp.kotlinLightSamples

fun testWhile()
{
    var count = 1

    while (count < 1000)
    {
        println("While count=>$count")
        count *= 2
    }
}

fun testDoWhile()
{
    var count = 1

    do
    {
        println("Do while count=>$count")
        count *= 2
    }
    while (count < 1000)
}
